data/scripts/molgenis-model-update/session.py

```diff
@@ -0,0 +1,151 @@
+import os
+import sys
+# import pathlib
+import requests
+
+
+class Session:
+    """
+
+    """
+
+    def __init__(self, url: str, database: str, email: str, password: str):
+        self.url = url
+        self.database = database
+        self.email = email
+        self.password = password
+        self.session = requests.Session()
+        self.graphqlEndpoint = self.url + '/' + self.database + '/graphql'
+        self.apiEndpoint = self.url + '/' + self.database + '/api'
+
+        self.sign_in()
+
+    def sign_in(self):
+        """Sign into molgenis and retrieve session cookie"""
+        query = 'mutation{signin(email: "%s", password: "%s"){status,message}}' % (self.email, self.password)
```

Comment:
For future: you might want to move to the 'token' feature of emx2 instead of user/password.
Then you don't need the whole signin stuff
But for now I propose you keep it so you can move forward